utils/node_local_test.py

```diff
@@ -2331,9 +2331,10 @@ def test_cont_rw(self):
         os.close(fd)
         print(ps)
 
-        with open(join(dfuse.dir, 'rw_dir', 'new_file'), 'r') as fd:
-            data = fd.read()
-            print(data)
+#       temporarily disable the test of reading file to avoid the issue from fuse kernel
```

Comment:
normally we also refer to DAOS-9671 in the comment

Reply:
Thank you! I will follow the convention next time.